Week1/world-insert.js

```diff
@@ -0,0 +1,58 @@
+const mysql = require('mysql');
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'world'
+});
+
+connection.connect();
+
+// insert rows
+const insert_query = [
+  "insert ignore into countries values(001, 'Brazil', 211243220 , 'South America', 8358140	)",
```

Comment:
Could you explain the use of "insert ignore" ? 
Is there a better way of avoiding the duplicate row ?